### PR TITLE
Update websphere-liberty license information

### PR DESCRIPTION
**Pull request by davidcurrie**
Restructured license.md to add information covering the fact that latest/8.5.5.5 image is now supported for limited capacity production usage (<2GB heap across all instances) in addition to development/build and provide links to information on building a fully production licensed image.

### CODE REVIEW

**Comment by davidcurrie** [comment]
Hmm - so I ran "markdownfmt.sh -l websphere-liberty" as per the instructions and it gave me a list of markdown files but no indication of what that meant. I guessing it meant it didn't like them!

File: websphere-liberty/README.md [hunk 1]
```diff
@@ -62,11 +62,15 @@ The image is designed to support a number of different usage patterns. The follo
...
+The Dockerfiles and associated scripts are licensed under the [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
+
+Licenses for the products installed within the images are as follows:* [IBM JRE](https://www14.software.ibm.com/cgi-bin/weblap/lap.pl?la_formnum=&li_formnum=L-EWOD-99YA4J&title=IBM%C2%AE+SDK%2C+Java+Technology+Edition%2C+Version+7+Release+1&l=en) (International License Agreement for Non-Warranted Programs)* [IBM WebSphere Application Server](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/8.5.5.5/lafiles/runtime/en.html) in the `latest`/`8.5.5` image (International License Agreement for Non-Warranted Programs)* [IBM WebSphere Application Server Liberty v9 Beta with Java EE 7](https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/beta/lafiles/en.html) in the `beta` image (International License Agreement for Early Release of Programs)
```
Comment (yosifkit) [inline websphere-liberty/README.md]: Were you trying to get a bulleted list here :wink:? (`s/[*] /\n-\t/`)

**Comment by davidcurrie** [comment]
Thanks for catching that. Took me half an hour to work out that it was actually the missing newline before the list that meant markdownfmt was squashing the list!

**Comment by yosifkit** [comment]
LGTM

**Comment by tianon** [comment]
LGTM

**Pull request merged**